testing/file_test/autoupdate.cpp

```diff
@@ -217,6 +217,22 @@ auto FileTestAutoupdater::AddRemappedNonCheckLine() -> void {
                    .second);
 }
 
+auto FileTestAutoupdater::AddTips() -> void {
+  CARBON_CHECK(tips_.empty()) << "Should only add tips once";
+
+  tips_.reserve(4);
+  // This puts commands on a single line so that they can be easily copied.
+  tips_.emplace_back("// TIP: To test this file alone, run:");
+  tips_.emplace_back("// TIP:   " + test_command_);
+  tips_.emplace_back("// TIP: To dump output, run:");
+  tips_.emplace_back("// TIP:   " + dump_command_);
```

Comment:
Only other thought I have here is if it makes sense to make these a bit easier to visually spot when looking at the files? I don't have many concrete ideas, and we can always revisit this, so not really suggesting a change here.

Reply:
For me, I find these easy to find, so I'm not quite sure what you'd be looking for.

Reply:
The lines just run together a bit for me skimming the file.

But this isn't important to solve here. Mostly mentioned in case you or others had ideas. Let's go ahead and land as-is.